src/test/java/com/google/devtools/build/lib/analysis/RunfilesRepoMappingManifestTest.java

```diff
@@ -435,4 +435,45 @@ public void overwriteWorkspaceFile(String... lines) throws Exception {
             .addAll(ImmutableList.copyOf(lines))
             .build());
   }
+
+  @Test
+  public void testInvisibleRepoInLabelResultsInEarlyError() throws Exception {
```

Comment:
I couldn't find a generic Bzlmod `BuildViewTestCase` test. Should I just create a new one?

Reply:
https://cs.opensource.google/bazel/bazel/+/master:src/test/java/com/google/devtools/build/lib/rules/LabelBuildSettingTest.java this is where I parked related tests